Travel Scheduler/View Controllers/ScheduleViewController.m

```diff
@@ -107,12 +108,15 @@ - (void)viewDidLoad
         self.endDate = getNextDate(self.startDate, 2);
     }
     [self scheduleViewSetup];
+    [self createGoToMapButton];
 }
 
 - (void)viewWillLayoutSubviews {
     [super viewWillLayoutSubviews];
     self.header.frame = CGRectMake(10, self.topLayoutGuide.length + 10, CGRectGetWidth(self.view.frame) - 10, 50);
     [self.header sizeToFit];
+    self.header.frame = CGRectMake(10, self.topLayoutGuide.length + 10, CGRectGetWidth(self.header.frame), CGRectGetHeight(self.header.frame));
```

Comment:
I think this line is unnecessary. You only need to set the frame twice when the x and y coordinates depend on the height and width of the label, but in this case, it doesn't so it's redundant.